### PR TITLE
Unselect image when CTRL + A

### PR DESCRIPTION
**Pull request by juliaroldi**
When a image is selected, if ctrl + a is pressed, unselect the image and select the whole content.

### CODE REVIEW

**Comment by JiuqingSong** [comment]
With this change I can't copy image using Ctrl+C

**Comment by JiuqingSong** [comment]
Suggest we unselect image according to selection change (browser selectionchange event) rather than the keyboard event

File: packages-content-model/roosterjs-content-model-core/lib/corePlugin/SelectionPlugin.ts [hunk 1]
```diff
@@ -221,6 +223,16 @@ class SelectionPlugin implements PluginWithState<SelectionPluginState> {
             if (newSelection?.type == 'range') {
                 this.state.selection = newSelection;
             }
```
Comment (JiuqingSong) [inline packages-content-model/roosterjs-content-model-core/lib/corePlugin/SelectionPlugin.ts:225]: Let's only do the original part for Safari, to reduce risk

File: packages-content-model/roosterjs-content-model-core/lib/corePlugin/SelectionPlugin.ts [hunk 2]
```diff
@@ -142,6 +143,7 @@ class SelectionPlugin implements PluginWithState<SelectionPluginState> {
     }
 
     private selectImage(editor: IEditor, image: HTMLImageElement) {
+        this.suspendSelectionChangeEvent = true;
         editor.setDOMSelection({
             type: 'image',
             image: image,
```
Comment (JiuqingSong) [inline packages-content-model/roosterjs-content-model-core/lib/corePlugin/SelectionPlugin.ts:149]: Can we set suspendSelectionChangeEvent = false after here?

Comment (juliaroldi) [reply packages-content-model/roosterjs-content-model-core/lib/corePlugin/SelectionPlugin.ts:149]: if we set false after here, it the selection will be removed when selectionchanged happened

File: packages-content-model/roosterjs-content-model-core/lib/corePlugin/SelectionPlugin.ts [hunk 3]
```diff
@@ -221,6 +223,18 @@ class SelectionPlugin implements PluginWithState<SelectionPluginState> {
             if (newSelection?.type == 'range') {
                 this.state.selection = newSelection;
             }
+
```
Comment (JiuqingSong) [inline packages-content-model/roosterjs-content-model-core/lib/corePlugin/SelectionPlugin.ts:226]: actually, I mean for line 223-225, it should only be run for Safari. Then for 227-235, it should be run for all

Comment (JiuqingSong) [reply packages-content-model/roosterjs-content-model-core/lib/corePlugin/SelectionPlugin.ts:226]: Please refer to comment at line 219

Comment (JiuqingSong) [reply packages-content-model/roosterjs-content-model-core/lib/corePlugin/SelectionPlugin.ts:226]: And this part should also not run when suspendSelectionChangeEvent.

Better add this check at beginning of this function

**Pull request closed without merging**